lpcompiler.pas

```diff
@@ -1459,7 +1459,7 @@ function TLapeCompiler.ParseMethod(FuncForwards: TLapeFuncForwards; FuncHeader:
     end;
 
     try
-      if (Tokenizer.Tok = tk_kw_Overload) or FuncHeader.isOperator then
+      if (Tokenizer.Tok = tk_kw_Overload) or (FuncHeader.isOperator and (Tokenizer.Tok <> tk_kw_Override)) then
```

Comment:
This seems like a weird statement? It will accept *any* keyword accept for `override`?